src/Cake.Npm/Cake.Npm.csproj

```diff
@@ -24,16 +24,8 @@
     <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net45|AnyCPU'">
-    <DocumentationFile>bin\Debug\net45\Cake.Npm.xml</DocumentationFile>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net45|AnyCPU'">
-    <DocumentationFile>bin\Release\net45\Cake.Npm.xml</DocumentationFile>
-  </PropertyGroup>
```

Comment:
I think these are still useful to have for net46 target. Or any reason why they are dropped?

Reply:
I just removed them as they referenced net45, could these be used to generate cakebuild.net docs?

Reply:
Yes, I think they are required for API documentation on cakebuild.net